frontend/src/v5/ui/routes/viewer/viewer.tsx

```diff
@@ -97,16 +100,16 @@ export const Viewer = () => {
 			model: containerOrFederation,
 			project,
 			teamspace,
-			revision,
+			revision: isFed ? undefined : revision,
```

Comment:
I think the check be 'revision === calibration' to be more explicit about this

Reply:
But what if someone names their revision "calibration"?

Reply:
that sucks, but fair enough. can you add a comment in the code to clarify the situation?